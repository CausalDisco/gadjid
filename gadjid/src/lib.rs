// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
