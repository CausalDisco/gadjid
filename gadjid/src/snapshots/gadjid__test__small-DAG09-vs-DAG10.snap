---
source: src/lib.rs
expression: "test(g_true, g_guess)"
---
g_true: 20009.DAG-10
g_guess: 20010.DAG-10
ancestor_aid:
  - 0.5666666666666667
  - 51
oset_aid:
  - 0.4777777777777778
  - 43
parent_aid:
  - 0.5777777777777777
  - 52
shd:
  - 0.3333333333333333
  - 15
t:
  - 0
  - 3
  - 8
y: 6
z:
  - 1
possible_descendants_of_t_in_g_guess:
  - 0
  - 1
  - 3
  - 4
  - 6
  - 8
not_amenable_in_g_guess_wrt_t: []
proper_ancestors_of_y_in_g_guess_wrt_t:
  - 1
  - 6
oset_for_t_onto_y_in_g_guess:
  - 3
not_validly_adjusted_for_in_g_guess_by_parents_of_t:
  - 1
  - 6
  - 8
not_validly_adjusted_for_in_g_guess_by_oset_for_t_onto_y:
  - 1
  - 3
  - 6
not_validly_adjusted_for_in_g_guess_by_empty_set:
  - 1
  - 6
not_validly_adjusted_for_in_g_guess_by_z:
  - 1
  - 6
